runtime/src/accounts_db.rs

```diff
@@ -308,12 +293,6 @@ impl IsCached for AccountInfo {
 
 impl IndexValue for AccountInfo {}
```

Comment:
Should this be moved over to the new file too?

Reply:
I was thinking these trait impls make sense here and they are not part of the implementation. I think of them is 'here's how to apply these traits to this ignorant data structure'